### PR TITLE
WIP hosted-loki: send all logs to Observatorium

### PR DESCRIPTION
**Pull request by vrutkovs**
Replace Grafana Cloud with Observatorium endpoint and credentials

### CODE REVIEW

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

File: ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh [hunk 1]
```diff
@@ -99,11 +98,9 @@ data:
           min_period: 1s
         batchsize: 102400
         batchwait: 10s
-        basic_auth:
-          username: ${GRAFANACLOUND_USERNAME}
-          password_file: /etc/promtail-grafanacom-secrets/password
+        bearer_token_file: /tmp/shared/prod_bearer_token
         timeout: 10s
-        url: https://logs-prod3.grafana.net/api/prom/push
+        url: ${LOKI_PROD_ENDPOINT}/push
```
Comment (dgoodwin) [inline ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:103]: Above this looks like it may be LOKI_ENDPOINT?

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade
/pj-rehearse pull-ci-openshift-app-netutil-master-e2e-aws

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade
/pj-rehearse pull-ci-openshift-app-netutil-master-e2e-aws

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade
/pj-rehearse pull-ci-openshift-app-netutil-master-e2e-aws

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

File: ci-operator/step-registry/ipi/install/rhobs-loki/ipi-install-rhobs-loki-commands.sh [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+        - mountPath: /etc/tls/private
+          name: proxy-tls
+        - mountPath: /etc/tls/cookie-secret
+          name: cookie-secret
+      - args:
+        - --oidc.client-id=\$(CLIENT_ID)
+        - --oidc.client-secret=\$(CLIENT_SECRET)
+        - --oidc.issuer-url=https://sso.redhat.com/auth/realms/redhat-external
+        - --margin=10m
+        - --file=/tmp/shared/prod_bearer_token
```
Comment (dgoodwin) [inline ci-operator/step-registry/ipi/install/rhobs-loki/ipi-install-rhobs-loki-commands.sh:426]: Is this mapped properly? I can't see a secret being mounted to this location.

Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/rhobs-loki/ipi-install-rhobs-loki-commands.sh:428]: Ah, right, its probably not being mounted in promtail container, fixed

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upgrade

**Pull request closed without merging**